.travis.yml

```diff
@@ -23,11 +23,7 @@ dist: trusty
 
 addons:
   firefox: latest
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
+  chrome: beta # stable version (60.0.3112.78) is bugged
```

Comment:
I'm not sure. I've updated the comment as I can't be positive if the bug is in our code or in Chrome. It is odd that Chart.js works with Chrome 59 and 61, but not 60. That was one reason I thought the issue was in Chrome. This only fixes the issue on Travis. Tests will still fail locally. But it will at least let us use Travis to evaluate incoming PRs.